OpenMEEG/src/assembleHeadMat.cpp

```diff
@@ -136,6 +136,7 @@ namespace OpenMEEG {
         }
     }
 
+    #if 0
```

Comment:
Should we remove this code then?

Reply:
Well it is work in progress, I intend to finish soon. So if we can keep it in, I prefer.